config/configuration_singleton.rb

```diff
@@ -94,4 +94,9 @@ def cluster_reload_delay
     cfg = ENV['OOD_CLUSTER_RELOAD_DELAY'].to_i
     cfg.positive? ? cfg : 1
   end
+
+  # Location of clusters configuration folder
+  def clusters_config_dir
+    ENV['CLUSTERS_CONFIG_DIR'] || '/etc/ood/config/clusters.d/'
```

Comment:
`OOD_CLUSTERS` is the env variable that other apps (like the dashboard) will respect.

```suggestion
    ENV['OOD_CLUSTERS'] || '/etc/ood/config/clusters.d/'
```